index.html

```diff
@@ -1,31 +1,28 @@
 <!DOCTYPE html>
-<html lang="ja">
+<html lang=“ja”>
   <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>プルリクエストがしたいよ！</title>
+    <meta charset=“UTF-8" />
+    <meta name=“viewport” content=“width=device-width, initial-scale=1.0" />
+    <meta http-equiv=“X-UA-Compatible” content=“ie=edge” />
+    <link rel=“stylesheet” href=“style.css” type=“text/css” />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
-    <p>プルリクエストがしたいよ！</p>
-    <div>
-      <ul id="List">
-      </ul>
-    </div>
-    <script>    
-      var ulElement = document.getElementById('List');
-      for(var i=1; i<=2; i++) {
-      var liElement = document.createElement('li');
-      liElement.textContent = 'a'+i;
-      ulElement.appendChild(liElement);
-      var aElement = document.createElement('a');
-      aElement.href= 'a'+i+'.html'
-      liElement.appendChild(aElement);
-      var imgElement = document.createElement('img');
-      imgElement.src = '/img/bookmark.png';
-      aElement.appendChild(imgElement);
-}
-    </script>
+      <div>
+        <ul id="js-list"></ul>
+      </div>
+  <script>
+      const ul = document.getElementById("js-list");
+      const li = document.createElement("li");
+      const a = document.createElement("a");
+      const img = document.createElement("img");
+      a.textContent = "これです";
+      a.setAttribute("href","1.html");
+      img.setAttribute("src", "bookmark.png");
+      img.setAttribute("alt","ブックマーク");
```

Comment:
setAttributeは非属性の時が使い所のようです。何かユーザー側で定義したい時

```suggestion
      a.href = "1.html"
      img.src = "bookmark.png"
      img.alt = "ブックマーク"
```

Reply:
参考URLいただきありがとうございます！